README.md

```diff
@@ -24,46 +24,82 @@ This section gives basic instructions to get Ursus running locally. More extensi
 
 ### Install and run locally
 
-First, install [Californica](https://github.com/UCLALibrary/californica) and ingest some data; make sure californica is running so ursus can point to its data.
+Ursus is a Black application and only needs Solr and Fedora.
+
+Ursus can be locally run in two ways:
+
+- Running in standalone mode
+- Running in conjunction with local instance Californica
+
+### Running in standalone mode
+
+The file `docker-compose-standalone.yml` includes a setup with a clone of the ursus-stage and sinai-stage solr indexes, so you do not need to run californica and manually ingest material (in fact, californica should #not# be running to avoid port conflicts.)
 
 Clone the repo from GitHub and change directories into the repo:
+
 ```
 git clone git@github.com:UCLALibrary/ursus.git
 cd ursus
 ```
 
 Set up the databases:
+
 ```
 docker-compose run web bundle exec rails db:setup
 docker-compose run sinai bundle exec rails db:setup
 ```
 
 Bring up the development environment. Do this _after_ setting up the databases - the startup scripts require the database to be ready so that they can set feature flags e.g. for the sinai UI mode.
+
 ```
 docker-compose up
 ```
 
 Ursus should now be running with its regular look on [port 3003](http://localhost:3003), and on [port 3004](http://localhost:3004) with the [Sinai Manuscripts Digital Library](https://sinaimanuscripts.library.ucla.edu/) UI enabled.
 
-### Running in standalone mode
+Note: to view Sinai images, first visit the [production site](https://sinaimanuscripts.library.ucla.edu) to load the cookie.
```

Comment:
Maybe for clarity this could say: Note: to view Sinai images, first visit the [production site](https://sinaimanuscripts.library.ucla.edu) and sign in/up to load the cookie.